templates/home.html

```diff
@@ -1,169 +1,51 @@
-}
-form input[type=email] {
-  border: 1px solid #ccc;
-  width: 73%;
-  font-size: 1.2em;
-}
-form input {
-  font: inherit;
-  padding: 15px 30px;
-  /*height: 60px;*/
-  display: inline;
-  vertical-align: middle;
-}
-form input.valid {
-  border: 1px solid #ccc;
-  background-color: rgba(243, 174, 115, .85);
-  color: #fff;
-  text-transform: uppercase;
-  /*width: 25%;*/
-  margin-left: 1%;
-  background-color: #6B5;
-}
-form input.valid[disabled] {
-  background-color: #999;
-  opacity: .6;
-}
-form input[type=checkbox] {
-  /*width: 35px;*/
-  margin-right: 1em;
-}
-form input[type=checkbox] + label {
-  font-size: 1.1em;
-}
-
-label:last-of-type {
-  display: inline-block;
-  margin-top: 1rem;
-  margin-bottom: 1.5rem;
-}
-
-#notice:not(.empty) {
-  padding: 20px;
-  font-weight: bold;
-  text-align: center;
-}
-
-@media screen and (max-width:960px) {
-  form input[type=email] {
-    width: 100%;
-  }
-  form input.valid {
-    display: block;
-    margin: 1.5rem auto 0;
-  }
-}
-
-@media screen and (max-width:620px) {
-  .earlybird {
-    left: auto;
-    width: 100%;
-  }
-  body > header > h1,
-  body > header > h2 {
-    left: 0;
-    width: 100%;
-    text-align: center;
-  }
-}
-
-</style>
-
-<header>
-  <section id=notice class=empty></section>
-  <img src={{ url_for('static', filename='homebackground.jpg') }} alt="Nomad worker">
-  <h1>N<span>e</span>omad</h1>
-  <h2>Make the world your home. And workplace.</h2>
-  <section class=earlybird>
-    <p>Do you want an early account? We are looking for <strong>beta users</strong>.</p>
-    <link href=//cdn-images.mailchimp.com/embedcode/classic-081711.css rel=stylesheet type=text/css>
-    <form method=post action=https://getsimpleform.com/messages?form_api_token=ec09216646c52506bc83161e4bc3e609>
-      <input type=hidden name=redirect_to value=http://neomad.org/#sent>
-      <input type=email name=email placeholder="Enter your email address" required>
-      <input class=valid type=submit value="Count me in!">
-      <label for=contribute>
-        <input type=checkbox id=contribute name=contribute>
-        I'd also like to contribute somehow! <small>(like: graphism, coding, ideas, …)</small>
-      </label>
-    </form>
-    <!-- <small><a href={{ url_for('signup') }}>I'm a newcomer</a></small> -->
-    <small><a href={{ url_for('login') }}>I already have an account</a></small>
+{% from 'blog/macros.html' import article_list with context %}
+
+{% extends 'layout.html' %}
+
+{% block title %}Digital Nomad Community{% endblock %} 
+{% block description %}Find a local place to work, get connected to other remote workers.{% endblock %}
+
+{% block extra_css %}
+  <link rel=stylesheet href={{ url_for('static', filename='css/home.css') }}>
+  <link rel=stylesheet href={{ url_for('static', filename='css/_article_list.css') }}>
+{% endblock %}
+
+{% block main %}
+  
+  <section>
+    <div>
+      <img src={{ url_for('static', filename='homebackground.jpg') }} alt="Nomad worker">
```

Comment:
On a semantic perspective, is this a content image (HTML) or a decorative background (CSS)?